sdk/include/opentelemetry/sdk/logs/batch_log_processor.h

```diff
@@ -41,11 +41,13 @@ struct BatchLogProcessorOptions
    */
   size_t max_export_batch_size = 512;
 
+#  ifdef ENABLE_ASYNC_EXPORT
   /**
    * Determines whether the export happens asynchronously.
    * Default implementation is synchronous.
    */
   bool is_export_async = false;
```

Comment:
Should we also remove `is_export_async_` in `BatchLogProcessor` and `BatchSpanProcessor` ?

Reply:
Done